packages/composables/src/useCustomerOrders/useCustomerOrders.ts

```diff
@@ -17,7 +17,9 @@ export type UseCustomerOrdersReturn = {
   /**
    * Fetches the orders list and assigns the result to the `orders` property
    */
-  loadOrders(parameters?: Schemas["Criteria"]): Promise<void>;
+  loadOrders(
+    parameters?: Schemas["Criteria"] & { checkPromotions?: boolean },
```

Comment:
it's already in the readOrder schema, what about
```suggestion
    parameters?: operations['readOrder post /order']['body'],
```